src/components/Avatar/Avatar.Crop.tsx

```diff
@@ -21,6 +21,7 @@ export const AvatarCrop = props => {
   )
 
   const styles = {
+    color: 'transparent',
```

Comment:
I think this transparent color needs to be applied only in the case where there is no image. See this example here https://deploy-preview-724--hsds-react.netlify.com/?path=/story/avatar--initials -- the initials are not visible due to the transparent background.

Reply:
Great catch. Fixed. 👍